bin/spoke

```diff
@@ -23,22 +26,43 @@ if (process.env.NODE_ENV === "development") {
 }
 
 const projectPath = commander.args.length > 0 ? commander.args[0] : process.cwd();
+const projectFilePath = path.join(projectPath, "spoke-project.json");
 
-console.log(`Starting ${pkg.name} server...`);
-console.log(`Serving project directory: ${projectPath}`);
+if (!fs.existsSync(projectFilePath)) {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+  });
 
-startServer({
-  projectPath,
-  port: commander.port,
-  https: commander.https
-})
-  .then(() => {
-    console.log(`Server listening on port ${commander.port}`);
+  rl.question("Creating new Spoke project. Would you like to add a the default assets? (y/n)", answer => {
+    const copyDefaultAssets = answer.toLowerCase() === "y";
 
-    if (commander.open) {
-      opn(`http://localhost:${commander.port}`);
-    }
-  })
-  .catch(error => {
-    console.error(error);
+    rl.close();
+
+    run(copyDefaultAssets);
   });
+} else {
+  run(false);
+}
+
+function run(copyDefaultAssets) {
+  console.log(`Starting ${pkg.name} server...`);
+  console.log(`Serving project directory: ${projectPath}`);
+
+  startServer({
+    projectPath,
+    copyDefaultAssets,
+    port: commander.port,
+    https: commander.https
+  })
+    .then(() => {
+      console.log(`Server listening on port ${commander.port}`);
+
+      if (commander.open) {
+        opn(`http://localhost:${commander.port}`);
+      }
```

Comment:
Maybe open by default, or ask here as well?

Reply:
Open by default seems like the correct thing to do.